notebook/notebookapp.py

```diff
@@ -53,13 +53,13 @@
 try:
     import tornado
 except ImportError:
-    raise ImportError(_("The Jupyter Notebook requires tornado >= 4.0"))
+    raise ImportError(_("The Jupyter Notebook requires tornado >= 5.0"))
 try:
     version_info = tornado.version_info
 except AttributeError:
-    raise ImportError(_("The Jupyter Notebook requires tornado >= 4.0, but you have < 1.1.0"))
+    raise ImportError(_("The Jupyter Notebook requires tornado >= 5.0, but you have < 1.1.0"))
 if version_info < (4,0):
```

Comment:
Please update the compared version.

Reply:
Oh, missed that one, thanks @kevin-bates

Reply:
Done (squashed new commit)